doc/system_health_monitoring/system-health-HLD.md

```diff
@@ -223,11 +282,19 @@ See open question 2 for adding configuration CLIs.
 
 ## 6. System health monitor test plan
 
-1. Kill some critical service and check the CLI output, the LED color and error shall be as expected.
-2. Simulate PSU/FAN and related sensor failure via mock sysfs and check the CLI output, the LED color and error shall be as expected.
-3. Change the monitor service/device list and restart the system health monitor service, then check whether the system health monitor service works as expected; also check whether the result of "show system-health monitor-list" aligned. 
+1. If some critical service missed, check the CLI output, the LED color and error shall be as expected.
+2. Simulate PSU/FAN/ASIC and related sensor failure via mock sysfs and check the CLI output, the LED color and error shall be as expected.
+3. Change the monitor service/device list then check whether the system health monitor service works as expected; also check whether the result of "show system-health monitor-list" aligned. 
 
 ## 7. Open Questions
-1. How to determine the SONiC system is in boot up stage ?
-2. Is it necessary to add a CLI to manipulate the configuration file? Like add/remove service/device from ignore list? 
+
+1. How to determine the SONiC system is in boot up stage? The current design is to compare the system up time with a "boot_timeout" value. The system up time is got from "cat /proc/uptime". The default "boot_timeout" is 300 seconds and can be configured by configuration. System health service will not do any check until SONiC system finish booting.
+
+```json
+{
+  "boot_timeout": 300
+}
+```
+
+2. Is it necessary to add a CLI to manipulate the configuration file? Like add/remove service/device from ignore list? This is not support for now.
```

Comment:
remove the second item, we don't want to support this CLI

Reply:
fixed